components/AvalancheDangerTable.tsx

```diff
@@ -22,12 +23,12 @@ export interface AvalancheDangerTableProps {
 export const AvalancheDangerTable: React.FunctionComponent<AvalancheDangerTableProps> = ({date, forecast, elevation_band_names, size}: AvalancheDangerTableProps) => {
   const {height, marginLeft, paddingTop} = {
     main: {
-      height: '200',
+      height: 200,
```

Comment:
NativeBase would silently convert these values to numbers - that seems like a bad practice for me. Now we validate and throw. `'200pt'` and `200` are valid values for React Native, `'200'` is not.